ert3/stats/_stats.py

```diff
@@ -1,17 +1,15 @@
 import numpy as np
 import scipy.stats
 
+from ert3.data import Record
+
 
 class Distribution:
     def __init__(self, *, size, index, rvs, ppf):
         if size is None and index is None:
-            raise ValueError(
-                "Cannot create distribution with neither size nor index"
-            )
+            raise ValueError("Cannot create distribution with neither size nor index")
         if size is not None and index is not None:
-            raise ValueError(
-                "Cannot create distribution with both size and index"
-            )
+            raise ValueError("Cannot create distribution with both size and index")
```

Comment:
Just curious, do you know why black decided to compress these lines all of a sudden?

Reply:
I think these files stopped being blacked at some point 🤔 This is however addressed CO wise in the function step PR of Dan 👍